packages/utils/src/TextUtils.ts

```diff
@@ -58,6 +58,24 @@ class TextUtils {
     }
     return 0;
   }
+
+  /**
+   * Outdent (opposite of indent) a string so the lowest indent level is 0
+   * Assumes each line is indented with the same characters (i.e. no mixed tabs/spaces)
+   * @param code The code to outdent
+   * @returns A code block where the minimum indent level of a line is 0
+   */
+  static outdentCode(code: string) {
```

Comment:
I'd almost rather put this in `ScriptEditorUtils`, as this function is more specific to a code block than it is to just plain text.